src/views/projects/Header.svelte

```diff
@@ -4,12 +4,13 @@
 
   import { config } from "@app/lib/config";
   import { pluralize } from "@app/lib/pluralize";
+  import { isLocal } from "@app/lib/utils";
 
   import CloneButton from "@app/views/projects/CloneButton.svelte";
   import Link from "@app/components/Link.svelte";
   import ProjectLink from "@app/components/ProjectLink.svelte";
   import SquareButton from "@app/components/SquareButton.svelte";
-  import { isLocal } from "@app/lib/utils";
+  import Icon from "@app/components/Icon.svelte";
```

Comment:
We sort the imports by the vim lexical sort, selecting the imports in visual mode and doing a `:sort` but you can also just sort it alphabetically here.

Reply:
[diff](https://github.com/radicle-dev/radicle-interface/compare/66d3e1496e2f5179e58be4af261c99494752b2d0..c3346575eece8c059370a629cea73470ae16ce78)